### ISSUE TITLE
normalize mean and std dev of images

### ISSUE THREAD

**Issue by manuelli** [RobotLocomotion/pytorch-dense-correspondence#55]
We need to think carefully how this might affect things if we later do batch-normalization
